proposals/graphql/proposal/src/main/asciidoc/chapters/about.asciidoc

```diff
@@ -37,27 +37,115 @@ Also see <<appendix_b>>
+* *flexible data retrieval*: by construction the query language enables to select the expected data in the response with a fine level of granularity,
+* *reduction of server requests*: the language allows the client to aggregate the expected data into a single request,
+* *easier version management*: thanks to the native capabilities to create new data while deprecating old ones, 
+* *partial results*: partial results are delivered by the GraphQL server in case of errors. A GraphQL result is made of data and errors. It's not an all or nothing approach. How to process such a result is the responsability of the client,
+* *low coupling with HTTP*: GraphQL does not try to make the most of the HTTP semantic. Queries can be made using GET or POST requests. The HTTP result code  does not reflect the GraphQL response. This aspect is subject to discussion and perhaps GraphQL will have to evolve towards a better integration with HTTP to broaden its adoption,
+* *challenging authorization handling*: an appropriate data access authorization policy must be defined and implemented to counter the extreme flexibility of the query language. For example, a customer may request data that are authorized and others that are prohibited,
+* *challenging API management*: most API management solutions are based on REST capabilities and allow for URL-based policies to be established. As we have seen, a GraphQL API has a single entry point. An analysis of the request is necessary to establish a policy. For instance, to prevent the execution of an overly complex request that would crash the server.
+
+[[graphql_beyond-the-hype]]
+=== GraphQL beyond the hype
+
+A lot has been said about GraphQL since its inception, creating noise and opposing believers vs skepticals. 
+
+We don't want to fuel the GraphQL vs REST controversy. We do not believe that GraphQL is a silver bullet and that it will replace REST. We do think that it is quite valid in some use cases and that it has a role to play in the world of modern APIs. 
+
+This opinion seems to us to be confirmed by:
+
+* the growth and dynamism of the  community, with, for example, events of international scope such as https://www.graphqlconf.org/[GraphQL Conf],
+* the adoption of the technology by major IT players (although it is difficult to have precise figures),
+* the recent creation of the https://gql.foundation/[GraphQL foundation] allowing for outreach beyond Facebook.
+
+[[graphql_and_database]]
+=== GraphQL and Databases
+
+GraphQL is about data manipulation and can be compared to SQL but this is a misconception. 
+
+GraphQL is not a database technology. It is a query language for APIs, not databases. It is database and storage agnostic and can be used in front of any kind of backend, with or without database.
+
+One of GraphQL's strength is its multi-datasource capability enabling to aggregate data from various sources with a single API.
+
+
+[[graphql_licensing]]
+=== GraphQL licensing
```

Comment:
Like above, I think the `GraphQL licensing` section is good information, but does not belong in the spec document.  If we need to make special accommodations for the GraphQL license, then we should mention the licenses involved and those accommodations but not necessarily reference the articles here.  That might be more appropriate in a separate document (like README or something linked from the README) or a separate blog post, etc.

Reply:
Agree